src/scanner/images/index.ts

```diff
@@ -115,13 +115,11 @@ export async function scanImages(
 
   for (const { imageName, fileSystemPath, imageWithDigest } of images) {
     try {
-      const shouldIncludeAppVulns = true;
       const archivePath = `docker-archive:${fileSystemPath}`;
 
       const pluginResponse = await scan({
         path: archivePath,
         imageNameAndTag: imageName,
-        'app-vulns': shouldIncludeAppVulns,
```

Comment:
will we still need this for node, java etc?

Reply:
[5.0.0](https://github.com/snyk/snyk-docker-plugin/releases/tag/v5.0.0) enables it by default!

Reply:
😱 great!